opentasks-provider/src/main/java/org/dmfs/provider/tasks/TaskDatabaseHelper.java

```diff
@@ -531,10 +531,10 @@ public final static String createIndexString(String table, boolean unique, Strin
         buffer.append("CREATE ");
         if (unique)
         {
-            buffer.append(" UNIQUE ");
+            buffer.append("UNIQUE ");
         }
-        buffer.append("INDEX ");
-        buffer.append(table).append("_").append(fields[0]).append("_idx ON ");
+		buffer.append("INDEX IF NOT EXISTS ");
+		buffer.append(table).append("_").append(fields[0]).append("_idx ON ");
```

Comment:
why the indentation now?

Reply:
I've squashed-rebased now to fix this, please check again. 

But finally it turned out what these look like identation changes are: the file has tabs instead of spaces and the IDE keeps them even though the code style uses spaces.

There is this settings:
![screenshot from 2017-11-29 14-39-18](https://user-images.githubusercontent.com/1077972/33378204-6088f8f0-d514-11e7-8753-59e7add96a11.png)

And this notification:
![screenshot from 2017-11-29 14-41-20](https://user-images.githubusercontent.com/1077972/33378218-6859e594-d514-11e7-9513-84c7f519ae0b.png)
(So in my commit those 2 lines were reformatted for some reason.)

I guess we could fix this in a separate ticket. Turn off that settings and re-format alll project files, to remove the tabs.